modules/rooms_booking_manager/rooms_booking_manager.availability_search.inc

```diff
@@ -225,8 +225,8 @@ function rooms_booking_availability_search_form_validate(&$form, &$form_state) {
  */
 function rooms_booking_availability_search_form_submit(&$form, &$form_state) {
 
-  $start_date = new DateTime($form_state['values']['rooms_start_date']);
-  $end_date = new DateTime($form_state['values']['rooms_end_date']);
+  $start_date = DateTime::createFromFormat($form['rooms_date_range']['rooms_start_date']['#date_format'], $form_state['input']['rooms_start_date']['date']);
+  $end_date = DateTime::createFromFormat($form['rooms_date_range']['rooms_end_date']['#date_format'], $form_state['input']['rooms_end_date']['date']);
```

Comment:
@plopesc because something formats the date in ['values'] as Y-m-d, but is not properly taking into account the format of the date that is input. (m/d/Y vs d/m/y) I had a good look around but could not figure out where, and decided to go with what is input since we know for certain what format it's in.